test/e2e/apimachinery/crd_publish_openapi.go

```diff
@@ -120,6 +126,11 @@ var _ = SIGDescribe("CustomResourcePublishOpenAPI [Privileged:ClusterAdmin]", fu
 			framework.Failf("%v", err)
 		}
 
+		ginkgo.By("kubectl explain works for CR with the same resource name as built-in object")
+		if err := verifyKubectlExplain(f.Namespace.Name, customServiceShortName+".spec", `(?s)DESCRIPTION:.*Specification of CustomService.*FIELDS:.*dummy.*<string>.*Dummy property`); err != nil {
```

Comment:
this modified a conformance test and introduced a test that has been flaking. This needs to be done in a non-conformance test first to demonstrate the test is stable.

Reply:
@liggitt should I remove this test from the conformance tests first?

Reply:
Yes, revert the conformance test to its previous state and isolate the new test in a non-conformance e2e test (marked as flaky until the test has been deflaked)